packages/replay/config/rollup.config.core.ts

```diff
@@ -1,6 +1,7 @@
 import replace from '@rollup/plugin-replace';
 import typescript from '@rollup/plugin-typescript';
 import { defineConfig } from 'rollup';
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
No action required, just putting this here for future reference: 

From the `@rollup/plugin-node-resolve` [documentation](https://www.npmjs.com/package/@rollup/plugin-node-resolve):

> resolveOnly: [...] An Array which instructs the plugin to limit module resolution to those whose names match patterns in the array. Note: Modules not matching any patterns will be marked as external

So iiuc, this is essentially syntactic sugar around adding everything to the `external` property in the rollup config except for the modules matching the patterns. I'm fine with using it as long as we can make it work with our rollup configs. In case that doesn't work, we at least know how to work around it.

Reply:
which is weird, as it wasn't marked as external before, right? But still. not included in the bundle 🤔 MAYBE we can drop the config option and only include `resolve()` (as it is also done in rollup.config.worker.ts), and _then_ it just takes the config from `external`... I'll give it a try!

Reply:
Update: Yes, that works. So I will remove the `resolveOnly` config, so it just uses the defined `externals`!